.changeset/bright-peas-destroy.md

```diff
@@ -0,0 +1,5 @@
+---
+"ctf/build-test-image": major
```

Comment:
we usually do a `minor` to start with

Reply:
@momentmaker so my version should be 1.0.0 but changeset should be minor?

Reply:
ah, it should be 0.0.0. OK then